### ISSUE TITLE
Mangayaro: No update extension after PR being merged

### ISSUE THREAD

**Issue by TheKingTermux** [tachiyomiorg/tachiyomi-extensions#19051]
### Source information

Mangayaro 1.4.27

### Source language

Indonesia

### Steps to reproduce

1. Open tachiyomi extension
2. Search update for mangayaro
3. The version of the extension is still 1.4.27 and no update after #19004 merged

### Expected behavior

The extension should update for new domain (https://mangayaro.id)

### Actual behavior

The extension still same with old domain (https://mangayaro.net) not the latest (https://mangayaro.id) one
Checked on the WebView 
![Screenshot_2023-11-21-01-26-36-04_afd66b7916d19a60815f56273408b73e](https://github.com/tachiyomiorg/tachiyomi-extensions/assets/50316075/5e3381c4-7eb7-42c1-b095-5a940aaa0a61)

### Tachiyomi version

0.14.7

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.7](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by vetleledaal** [comment]
The CI was failing, this was addressed by #19048 which completed just now. Should be available in the app in 10-15 minutes due to caching.

**Comment by TheKingTermux** [comment]
> The CI was failing, this was addressed by #19048 which completed just now. Should be available in the app in 10-15 minutes due to caching.

Glad to hear that, thanks for the confirmation
